adserver/tasks.py

```diff
@@ -35,6 +35,8 @@ def daily_update_geos(day=None):
         queryset = Offer.objects.filter(
             date__gte=start_date,
             date__lt=end_date,  # Things at UTC midnight should count towards tomorrow
+            # TODO: Start indexing null offers. We need a new name for them -- maybe `requests` or `decisions`?
```

Comment:
I agree this isn't strictly necessary with this PR but it will be nice. Are you thinking these would go into a separate table? I think decision is good as a name if that's the question.

Reply:
Not really sure what the best option is. I think it's natural to add it to the Offer `offered/viewed/clicked`, but that migration seems scarier :)

I think for now a secondary index and report, eg. "Reports > Fill Rate", and then we can adjust the modeling if we want to at some point, but we have all the data we need for now.